### PR TITLE
fix(TBD-12079):Support Cluster Policy for Databricks transient clusters

### PR DESCRIPTION
**Pull request by cbadillo1603**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.dbr550/plugin.xml [hunk 1]
```diff
@@ -21,8 +21,8 @@
         <libraryNeeded
             context="plugin:org.talend.hadoop.distribution.dbr550"
             id="talend-bigdata-launcher-databricks-2.0.14"
```
Comment (hliashchuk) [inline main/plugins/org.talend.hadoop.distribution.dbr550/plugin.xml:23]: I guess you forgot change the number in id?

Comment (cbadillo1603) [reply main/plugins/org.talend.hadoop.distribution.dbr550/plugin.xml:23]: Yes I'll update it

**Review by hliashchuk — DISMISSED** [review]

File: main/plugins/org.talend.hadoop.distribution.dbr550/plugin.xml [hunk 2]
```diff
@@ -277,7 +277,11 @@
                 description="Big Data launcher libraries for DBR550"
                 id="BIGDATA-LAUNCHER-LIB-DBR550"
                 name="BIGDATA-LAUNCHER-LIB-DBR550"  >
+<<<<<<< HEAD
```
Comment (hliashchuk) [inline main/plugins/org.talend.hadoop.distribution.dbr550/plugin.xml:280]: Bad conflicts resolving :)

**Review by hliashchuk — CHANGES_REQUESTED** [review]

**Comment by hliashchuk** [comment]
the main/plugins/org.talend.libraries.*/.project files don't belong here, I guess

**Comment by hliashchuk** [comment]
otherwise lgtm

**Comment by cbadillo1603** [comment]
Changes done, please review.

**Review by hliashchuk — APPROVED** [review]

**Review by lbourgeois — COMMENTED** [review]
I would have moved to 2.1.0 as it is a new functionality and to follow https://semver.org/

**Review by AlixMetivier — APPROVED** [review]

**Review by lbourgeois — APPROVED** [review]
LGTM

**Pull request merged**